core/models/classifiers.go

```diff
@@ -3,6 +3,8 @@ package models
 import (
 	"context"
 	"database/sql/driver"
+	"github.com/greatnonprofits-nfp/goflow/services/classification/dialogflowcl"
```

Comment:
I think you need to update the GoFlow library here on Mailroom referencing the latest commit from there